### DISCUSSION TITLE
WixToolset and maven updates

### DISCUSSION THREAD

**Discussion by yura-kit** [wixtoolset/issues#8146] [Questions]
Hi, we are currently using org.wixtoolset:wix-toolset from the Maven repository. 
Due to a new vulnerability, how can we switch to version 4.0.4? 
It seems that version 4.0.4 is not available in the Maven repository.
https://mvnrepository.com/artifact/org.wixtoolset/wix-toolset

**Comment by robmen** [comment]
You'll need to talk to the keepers of the Maven repository. We don't maintain those.

**Comment by chrpai** [comment]
If you look at the 3.11 version you'll see a couple Maven devs (one with an email)  who you probably want to talk to.  FWIW,  wix  4/5 probably doesn't need to be in maven since really all you need to do is use dotnet to install them.  The latest 3.14.1 is what you probably want to ask them to include.  It's not radically different like 4/5 is so it should be easier for them to oblige you and much less backwards compatibility changes.

**Reply by yura-kit** [reply]
Does the latest 3.14.1 contains fix for CVE-2024-24810?
Thank you for your quick answer!

**Reply by chrpai** [reply]
If you read the link you posted you'll see it was fixed in 3.14.0 so yes 3.14.1 contains it also.
